applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -872,6 +848,29 @@ bool verify_datamode_control(uint16_t time_limit, uint16_t *min_time_limit)
 	return true;
 }
 
+struct at_param_list *slm_get_at_param_list(const char *at_cmd)
+{
+	int err;
+
+	at_params_list_clear(&slm_at_param_list);
+	err = at_parser_params_from_str(at_cmd, NULL, &slm_at_param_list);
+	if (err) {
+		LOG_ERR("AT command parsing failed: %d", err);
+		at_params_list_clear(&slm_at_param_list);
+	}
+
+	return &slm_at_param_list;
+}
+
+void set_default_at_response(void *buf, size_t size)
+{
+	const int err = at_cmd_custom_respond(buf, size, "OK\r\n");
+
+	if (err) {
+		LOG_ERR("Set default response failed: %d", err);
+	}
+}
```

Comment:
I wonder if we should actually return the error from here.

Also, most commands seem to call this first, then might return something. Is this right?
Examples seem to call this last https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/modem/at_cmd_custom.html#c.at_cmd_custom_respond

Reply:
Weird, why `buf+2`? Well, good that this gets resolved by the macro approach. Even if it doesn't currently affect the AT commands we have, I wouldn't want to take a chance on this biting us in the future.

Reply:
`nrf_modem_at_cmd` leaves space for `\r\n` in the response buffer.